### DISCUSSION TITLE
Introducing AneUrlLoader: A Robust Alternative to Adobe AIR's URLLoader

### DISCUSSION THREAD

**Discussion by bobaoapae** [airsdk/Adobe-Runtime-Support#3436] [Show and tell]
AneUrlLoader is a powerful Adobe AIR Native Extension (ANE) that enhances the capabilities of the standard URLLoader. Unlike the native AIR URLLoader, AneUrlLoader supports advanced networking features such as Happy Eyeballs (RFC 8305) for improved connection performance, HTTP/2 for faster data transfer, and TLS 1.3 for enhanced security. It also includes a custom DNS resolver, giving you control over which DNS servers to use, with default support for Cloudflare and Google. These features make AneUrlLoader a superior choice for developers looking for greater flexibility, security, and performance in their network operations across Windows 32-bit, Android, and macOS platforms. If you need a more efficient and customizable solution for loading URLs in your AIR applications, AneUrlLoader is the right tool for you.

https://github.com/bobaoapae/ane-urlloader

**Comment by marchbold** [comment]
Nice work!!

**Comment by ylazy** [comment]
Nice! Thank you!

**Comment by bobaoapae** [comment]
Initial ios support added.
